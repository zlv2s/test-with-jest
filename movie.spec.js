import movie from './movie'

describe('favorite movie', () => {
  let myMovie = []
  beforeEach(() => {
    myMovie = [{
      title: 'Age of Ultron',
      rate: null
    }]
  })

  test.only('can add a movie', () => {
    movie.add(myMovie, 'kungfu panda')
    expect(myMovie).toMatchSnapshot()
  })

  test('rate a movie', () => {
    movie.rate(myMovie[0], 5)
    expect(myMovie).toMatchSnapshot()
  })
})