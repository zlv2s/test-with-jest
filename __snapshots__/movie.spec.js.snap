// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`favorite movie can add a movie 1`] = `
Array [
  Object {
    "rate": null,
    "title": "Age of Ultron",
  },
  Object {
    "rate": null,
    "title": "kungfu panda",
  },
]
`;

exports[`favorite movie rate a movie 1`] = `
Array [
  Object {
    "rate": 5,
    "title": "Age of Ultron",
  },
]
`;
